### PR TITLE
Remove feature flag for remove decision reviews.

### PR DESCRIPTION
**Pull request by Sjones352**
Connects #10658

### CODE REVIEW

**Comment by Sjones352** [comment]
@pkarman It's been a month since we enabled `remove decision review` flag. Can you please review pr when you get a chance

File: client/app/intakeEdit/components/EditButtons.jsx [hunk 1]
```diff
@@ -126,10 +113,6 @@ class SaveButtonUnconnected extends React.Component {
 
     const saveButtonText = withdrawReview ? 'Withdraw' : 'Save';
 
-    const removeVbmsCopy = 'This will remove the review and cancel all the End Products associated with it.';
```
Comment (pkarman) [inline client/app/intakeEdit/components/EditButtons.jsx]: why are we also deleting this confirmation message?

File: spec/feature/intake/appeal/edit_spec.rb [hunk 2]
```diff
@@ -723,8 +711,6 @@
           end
 
           click_edit_submit
-          expect(page).to have_content("Remove review?")
```
Comment (pkarman) [inline spec/feature/intake/appeal/edit_spec.rb]: doesn't this confirmation message need to stay?

File: spec/feature/intake/higher_level_review/edit_spec.rb [hunk 3]
```diff
@@ -1343,8 +1330,6 @@ def click_cancel(visit_page)
         end
 
         click_edit_submit
-        expect(page).to have_content("Remove review?")
```
Comment (pkarman) [inline spec/feature/intake/higher_level_review/edit_spec.rb]: keep this?

File: spec/feature/intake/higher_level_review/edit_spec.rb [hunk 4]
```diff
@@ -1375,8 +1360,6 @@ def click_cancel(visit_page)
...
```
Comment (pkarman) [inline spec/feature/intake/higher_level_review/edit_spec.rb]: keep this?

File: client/app/intakeEdit/components/EditButtons.jsx [hunk 5]
```diff
@@ -21,8 +21,7 @@ class SaveButtonUnconnected extends React.Component {
       originalIssueNumber: this.props.state.addedIssues.length,
       showModals: {
         issueChangeModal: false,
-        unidentifiedIssueModal: false,
-        reviewRemovedModal: false
```
Comment (leikkisa) [inline client/app/intakeEdit/components/EditButtons.jsx]: Following up on Peter's comments, this should stay here as well.

File: spec/feature/intake/appeal/edit_spec.rb (hunk 2 above)
Comment (leikkisa) [reply spec/feature/intake/appeal/edit_spec.rb]: ➕

File: spec/feature/intake/higher_level_review/edit_spec.rb (hunk 3 above)
Comment (leikkisa) [reply spec/feature/intake/higher_level_review/edit_spec.rb]: ➕

File: client/app/intakeEdit/components/EditButtons.jsx [hunk 6]
```diff
@@ -144,17 +127,6 @@ class SaveButtonUnconnected extends React.Component {
...
-        title="Remove review?"
-        buttonText= "Yes, remove"
-        onClose={() => this.closeModal('reviewRemovedModal')}
-        onConfirm={() => this.confirmModal('reviewRemovedModal')}>
-        <p>
-          {originalIssueNumberCopy}
-        </p>
-        <p>{processedInCaseflow ? removeCaseflowCopy : removeVbmsCopy}</p>
-      </SaveAlertConfirmModal>}
-
```
Comment (leikkisa) [inline client/app/intakeEdit/components/EditButtons.jsx]: All of this will stay

File: client/app/intakeEdit/components/EditButtons.jsx [hunk 7]
```diff
@@ -91,21 +85,14 @@ class SaveButtonUnconnected extends React.Component {
...
       veteranValid,
-      processedInCaseflow,
       withdrawalDate,
       receiptDate
     } = this.props;
 
     let disableDueToIssueCount = false;
 
-    if (!issueCount && !removeDecisionReviews) {
-      disableDueToIssueCount = true;
```
Comment (leikkisa) [inline client/app/intakeEdit/components/EditButtons.jsx]: This stays, but you can remove !removeDecisionReviews

File: client/app/intakeEdit/components/EditButtons.jsx [hunk 8]
```diff
@@ -31,16 +30,11 @@ class SaveButtonUnconnected extends React.Component {
...
-      reviewRemovedModal: false
+      unidentifiedIssueModal: false
     };
 
     if (this.state.originalIssueNumber !== this.props.state.addedIssues.length) {
-      if (this.props.state.addedIssues.length === 0 && this.props.removeDecisionReviews) {
-        showModals.reviewRemovedModal = true;
-      } else {
-        showModals.issueChangeModal = true;
-      }
```
Comment (leikkisa) [inline client/app/intakeEdit/components/EditButtons.jsx]: This section should stay, but you don't need to check this.props.removeDecisionReviews

File: client/app/intake/components/ErrorAlert.jsx [hunk 9]
```diff
@@ -41,10 +41,6 @@ export default class ErrorAlert extends React.PureComponent {
         body: `An EP ${this.props.errorData} for this Veteran's claim was created` +
               ' outside Caseflow. Please tell your manager as soon as possible so they can resolve the issue.'
       },
-      request_issues_data_empty: {
-        title: 'No issues were selected',
-        body: 'Please select at least one issue and try again.'
-      },
```
Comment (leikkisa) [inline client/app/intake/components/ErrorAlert.jsx]: We don't need this on edit, but you might want to confirm if we would use this on intake.

File: client/app/intakeEdit/components/EditButtons.jsx (hunk 8 above)
Comment (Sjones352) [reply client/app/intakeEdit/components/EditButtons.jsx]: Fixed

File: client/app/intakeEdit/components/EditButtons.jsx (hunk 7 above)
Comment (Sjones352) [reply client/app/intakeEdit/components/EditButtons.jsx]: Fixed

File: client/app/intakeEdit/components/EditButtons.jsx (hunk 6 above)
Comment (Sjones352) [reply client/app/intakeEdit/components/EditButtons.jsx]: Fixed

File: spec/feature/intake/higher_level_review/edit_spec.rb (hunk 4 above)
Comment (Sjones352) [reply spec/feature/intake/higher_level_review/edit_spec.rb]: Fixed

File: spec/feature/intake/appeal/edit_spec.rb (hunk 2 above)
Comment (Sjones352) [reply spec/feature/intake/appeal/edit_spec.rb]: Fixed

File: client/app/intakeEdit/components/EditButtons.jsx (hunk 1 above)
Comment (Sjones352) [reply client/app/intakeEdit/components/EditButtons.jsx]: Sorry I thought we were removing the remove-decision modal. It's now fixed and ready for review.

**Comment by Sjones352** [comment]
@pkarman I would like to merge this but it is waiting for your approval.

**Pull request merged**